packages/classic-webview/src/components/header.tsx

```diff
@@ -45,7 +45,10 @@ export const Header = () => {
           ) : (
             <>
               <Logo />
-              <div className="flex gap-1">
+              <div
```

Comment:
This should be a button!
```suggestion
              <button
```